server/tests/api/video-blacklist-management.js

```diff
@@ -0,0 +1,235 @@
+
+	done()
+      })
+    })
+
+    it('Should get the correct sort when sorting by descending id', function (done) {
+      videoBlacklistsUtils.getSortedBlacklistedVideosList(servers[0].url, servers[0].accessToken, '-id', function (err, res) {
+        if (err) throw err
+
+        expect(res.body.total).to.equal(2)
+
+        const videos = res.body.data
+        expect(videos).to.be.an('array')
+        expect(videos.length).to.equal(2)
+
+        const result = res.body.data.slice(0).sort((a, b) => {
+          if (a.id > b.id) return -1
+          if (a.id < b.id) return 1
+          return 0
+        })
+
+        expect(videos).to.deep.equal(result)
+
+        done()
+      })
+    })
+
+    it('Should get the correct sort when sorting by descending video name', function (done) {
+      videoBlacklistsUtils.getSortedBlacklistedVideosList(servers[0].url, servers[0].accessToken, '-name', function (err, res) {
+        if (err) throw err
+
+        expect(res.body.total).to.equal(2)
+
+        const videos = res.body.data
+        expect(videos).to.be.an('array')
+        expect(videos.length).to.equal(2)
+
+        const result = res.body.data.slice(0).sort((a, b) => {
+          if (a.name > b.name) return -1
+          if (a.name < b.name) return 1
+          return 0
+        })
+
+        expect(videos).to.deep.equal(result)
+
+        done()
+      })
+    })
+
+    it('Should get the correct sort when sorting by ascending creation date', function (done) {
+      videoBlacklistsUtils.getSortedBlacklistedVideosList(servers[0].url, servers[0].accessToken, 'createdAt', function (err, res) {
+        if (err) throw err
+
+        expect(res.body.total).to.equal(2)
+
+        const videos = res.body.data
+        expect(videos).to.be.an('array')
+        expect(videos.length).to.equal(2)
+
+        const result = res.body.data.slice(0).sort((a, b) => {
+          if (a.createdAt < b.createdAt) return -1
+          if (a.createdAt > b.createdAt) return 1
+          return 0
+        })
+
+        expect(videos).to.deep.equal(result)
+
+        done()
+      })
+    })
+  })
+
+  describe('When removing a blacklisted video', function () {
+    it('Should not have any video in videos search on pod 1', function (done) {
+      videosUtils.getVideosList(servers[0].url, function (err, res) {
+        if (err) throw err
+
+        expect(res.body.total).to.equal(0)
+        expect(res.body.data).to.be.an('array')
+        expect(res.body.data.length).to.equal(0)
+
+        done()
+      })
+    })
+
+    it('Should remove a video from the blacklist on pod 1', function (done) {
+      series([
+        // Get one video in the blacklist
+	function (next) {
+          videoBlacklistsUtils.getSortedBlacklistedVideosList(servers[0].url, servers[0].accessToken, '-name', function (err, res) {
+	    if (err) throw err
+
+	    servers[0].videoToRemove = res.body.data[0]
+	    servers[0].blacklist = res.body.data.splice(1)
```

Comment:
Why splice?

Reply:
I think it's a mistake. It should be `slice(1)`. I will correct it ;)